packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -366,6 +375,7 @@ abstract class ResidentRunner {
 
   void printHelpDetails() {
     if (supportsServiceProtocol) {
+      printStatus('To detach from the application without ending it, press "d".');
```

Comment:
I'm not sure we want to show this to every Flutter developer. Maybe make this line optional depending on Fuchsia?